### DISCUSSION TITLE
Chrome ralated

### DISCUSSION THREAD

**Discussion by KhineHbawi** [community/community#132516] [Repositories]
how to set up

**Comment by queenofcorgis** [comment]
Hi @KhineHbawi ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Google Chrome. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by KhineHbawi** [comment]
132516
